enhancements/olm/olm-managed-operator-metrics.md

```diff
@@ -103,19 +99,54 @@ A `ServiceMonitor` object must be created in the namespace mentioned above and p
+The OpenShift Monitoring Requirements mentioned above can be met in one of two ways:
 
-OLM [building-your-csv](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md) documentation will be updated to describe two new supported annotations:
+- By using annotations that instructs OpenShift Console to generate the required Namespace and RBAC Requirements
+- By packaging the RBAC [mentioned above](#####rbac-requirements) within the Operator Bundle and having a cluster admin install the operator in a namespace that is both prefixed with `openshift-` and has the `openshift.io/cluster-monitoring=true` label.
 
-- The `operatorframework.io/suggested-namespace` annotation. When the `operatorframework.io/suggested-namespace` annotation is present the UI will highlight the suggested namespace when installing the operator. OLM itself will not require that the operator is deployed in the namespace defined by the `operatorframework.io/suggested-namespace` annotation.
+##### Fulfilling the Namespace and RBAC Requirements via Console
 
-- The `operatorframework.io/cluster-monitoring=true` annotation. When this annotation is set to `true`, the OpenShift Console will update the namespace that the operator is being deployed to with the `openshift.io/cluster-monitoring=true` label. When this annotation is present, the UI will update the OpenShift Monitoring Prometheus Operator ServiceAccount with the appropriate RBAC privileges for the given namespace as well, allowing operators to be scraped by the OpenShift Monitoring Prometheus Operator.
+OLM  [building-your-csv](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md) documentation will be updated to describe two new supported annotations:
 
->Note: There is no work required by the OLM team to implement this support.
+- The `operatorframework.io/suggested-namespace` annotation. When the `operatorframework.io/suggested-namespace` annotation is present the UI will highlight the suggested namespace when installing the operator. OLM itself will not require that the operator is deployed in the namespace defined by the `operatorframework.io/suggested-namespace` annotation.
+- The `operatorframework.io/cluster-monitoring=true` annotation. When this annotation is set to `true`, the OpenShift Console will update the namespace that the operator is being deployed to with the `openshift.io/cluster-monitoring=true` label. When this annotation is present, the UI will update the OpenShift Monitoring `prometheus-k8s` ServiceAccount with the [required RBAC](#####rbac-requirements) privileges for the given namespace as well, allowing operators to be scraped by the OpenShift Monitoring Prometheus instance.
```

Comment:
@awgreene is this available outside dev console. We are moving our service monitors in olm here https://github.com/openshift-knative/serverless-operator/pull/761 and we are wondering if rbac automatic setup can happen when we install an operator via cli.

Reply:
Hello @skonto! This is currently an console only feature - OLM does not create or update the namespace with the required annotation or update the RBAC of the prometheus-k8s ServiceAccount. This was done intentionally to ensure that Cluster Admins are given the opportunity to chose which operators are given permission to report metrics to Telemeter.

I can think of ways that you could circumvent this limitation, but I suspect that there could be an issue where a customer might get upset if they are unknowingly reporting data from your operator to Red Hat.